static/docs/get-started/reusing-data.md

```diff
@@ -0,0 +1,48 @@
+# Reusing data
+
+We've explained how to [upload](/doc/get-started/share-data) and
+[download](/doc/get-started/retrieve-data) <abbr>data artifacts</abbr> (with
+`dvc pull` and `dvc push`) as a way to share them on remote storage. This is
+great for several people to work on the same <abbr>DVC project</abbr> (on
+different machines). But what if we wanted to reuse a dataset from a project in
+a completely different one?
+
+That's why we created `dvc import`! This commands can download data from any DVC
+project hosted on a Git repository. It also tracks the imported data by creating
+an import stage ([DVC-file](/doc/user-guide/dvc-file-format)). Example:
+
+```dvc
+$ dvc import https://github.com/iterative/dataset-registry \
+           get-started/data.xml
+```
+
+> This command is only shown for informational purposes. No need to actually run
```

Comment:
may be we can say something like "Alternatively, get started could have been initialized with `dvc import` to ..." and actually show that path (remove dvc-file and data/data.xml, etc)

Reply:
Previously addressed.